services/app-api/prisma/schema.prisma

```diff
@@ -175,62 +175,47 @@ model SharedRateCertifications {
-  rateWithCertifyingActuaryID    String?
-  rateWithAddtlActuaryID    String?
-  rateActuaryCertifying  RateRevisionTable?  @relation(name: "CertifyingActuaryOnRateRevision", fields: [  rateWithCertifyingActuaryID ], references: [id])
-  rateActuaryAddtl   RateRevisionTable?   @relation(name: "AddtlActuaryOnRateRevision", fields: [   rateWithAddtlActuaryID ], references: [id])
+  id                          String             @id @default(uuid())
+  createdAt                   DateTime           @default(now())
+  updatedAt                   DateTime           @default(now()) @updatedAt
+  name                        String?
+  titleRole                   String?
+  email                       String?
+  actuarialFirm               ActuarialFirm?
+  actuarialFirmOther          String?
+  rateWithCertifyingActuaryID String?
+  rateWithAddtlActuaryID      String?
+  rateActuaryCertifying       RateRevisionTable? @relation(name: "CertifyingActuaryOnRateRevision", fields: [rateWithCertifyingActuaryID], references: [id])
+  rateActuaryAddtl            RateRevisionTable? @relation(name: "AddtlActuaryOnRateRevision", fields: [rateWithAddtlActuaryID], references: [id])
+  display_seq                 Int?
 }
 
 model ContractDocument {
-  id                 String                @id @default(uuid())
-  createdAt          DateTime              @default(now())
-  updatedAt          DateTime              @default(now()) @updatedAt
-  name               String
-  s3URL              String
-  sha256             String
-  contractRevisionID String
-  contractRevision   ContractRevisionTable @relation(fields: [contractRevisionID], references: [id])
-}
-
-model ContractSupportingDocument {
-  id                 String                @id @default(uuid())
-  createdAt          DateTime              @default(now())
-  updatedAt          DateTime              @default(now()) @updatedAt
-  name               String
-  s3URL              String
-  sha256             String
-  contractRevisionID String
-  contractRevision   ContractRevisionTable @relation(fields: [contractRevisionID], references: [id])
+  id                                 String                 @id @default(uuid())
+  createdAt                          DateTime               @default(now())
+  updatedAt                          DateTime               @default(now()) @updatedAt
+  name                               String
+  s3URL                              String
+  sha256                             String
+  contractDocumentRevisionID         String
+  supportingDocumentRevisionID       String
+  contractDocumentContractRevision   ContractRevisionTable? @relation(name: "ContractDocumentOnContractRevision", fields: [contractDocumentRevisionID], references: [id])
+  supportingDocumentContractRevision ContractRevisionTable? @relation(name: "SupportingDocumentOnContractRevision", fields: [supportingDocumentRevisionID], references: [id])
+  display_seq                        Int?
```

Comment:
Seems a bit dodgy to have this be optional, when would we not want to set it? I think all of these things need to have a visual order every time

Reply:
@macrael The reason i didn't set this is we haven't discussed when and how it gets set. I was just adding it in to be used in the future since we discussed it in slack.  Is it set in handlers I'm guessing - so on create or update prisma actions?  And then we start to `orderBy` that in the queries? Let me know if that's right --  I will implement!